addons/udes_ldap/views/res_company_ldap_views.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <menuitem id="udes_ldap.ldap_configuration"
+      parent="base.menu_users"
+      name="LDAP Configuration"
+    />
+
+    <menuitem id="udes_ldap.ldap_configuration_action"
+      parent="base.menu_users"
+      action="auth_ldap.action_ldap_installer"
+      sequence="40"
+      groups=""
+      name="LDAP Servers"
+    />
+
+    <record id="view_ldap_installer_form" model="ir.ui.view">
+        <field name="name">udes_ldap.res.company.ldap.form</field>
+        <field name="model">res.company.ldap</field>
+        <field name="inherit_id" ref="auth_ldaps.view_ldap_installer_form"/>
+        <field name="arch" type="xml">
+            <!-- Hide inherited fields for login informtion. -->
+          <xpath expr="//field[@name='ldap_binddn']" position="attributes">
+              <attribute name="invisible">1</attribute>
```

Comment:
Do we want it/them invisible or just remove it completely from the view?

Reply:
Removed